docs/contributing/running-tests.md

```diff
@@ -38,6 +38,20 @@ To run these tests locally, add `-PtestLatestDeps=true` to your existing `gradle
 
 Executing `./gradlew :instrumentation:<INSTRUMENTATION_NAME>:test --tests <GROOVY TEST FILE NAME>` will run only the selected test.
 
+### Debugging tests
+
+If you have trouble debugging tests in IntelliJ IDEA (because the classpath is not correct),
+you can use remote debugging:
+
+1. Run the tests with the `--debug-jvm` flag.
```

Comment:
[minor] maybe link to gradle documentation about this.

Reply:
is this https://docs.gradle.org/current/userguide/java_testing.html#sec:debugging_java_tests?